test/viewport.js

```diff
@@ -60,9 +60,24 @@ test('bounds for 512px tiles', function (t) {
     var xMax = bounds[2];
     var yMax = bounds[3];
 
-    t.equal(precisionRound(xMin, decDegreesFloatTolerance), precisionRound(-77.03945338726044, decDegreesFloatTolerance));
-    t.equal(precisionRound(yMin, decDegreesFloatTolerance), precisionRound(38.89697827424865, decDegreesFloatTolerance));
-    t.equal(precisionRound(xMax, decDegreesFloatTolerance), precisionRound(-77.03365981578827, decDegreesFloatTolerance));
-    t.equal(precisionRound(yMax, decDegreesFloatTolerance), precisionRound(38.89843950894583, decDegreesFloatTolerance));
+    t.equal(precisionRound(xMin, decDegreesFloatTolerance), -77.03945339);
+    t.equal(precisionRound(yMin, decDegreesFloatTolerance), 38.89697827);
+    t.equal(precisionRound(xMax, decDegreesFloatTolerance), -77.03365982);
+    t.equal(precisionRound(yMax, decDegreesFloatTolerance), 38.89843951);
     t.end();
 });
+
+test('bounds for float zooms', function (t) {
```

Comment:
If you're wondering where the magic values came from, in these test expectations...

I didn't have the necessary background in the `node-sphericalmercator` math to create my expectations with math, so I fell back to good old GIS. 

My original use case for this module was creating GeoJSON bounding boxes that bound Mapbox Static Map API images. So, I used this as my reality check. I'd use the API to request an image, use `bounds()` to determine its bbox, then use QGIS to do a visual check.

To that end, it's slightly imprecise, but matches my use case.

Reply:
I think this is fine. One thing to do would be add a test for z `16` and z `16.5` and assert that the values are less/greater than each other, essentially asserting they output _different_ values.

No need to do this now, though!